pkg/apis/knative/v1alpha1/deployer_validation.go

```diff
@@ -88,6 +88,19 @@ func (s DeployerSpec) Validate() validation.FieldErrors {
 		errs = errs.Also(validation.ErrInvalidValue(s.IngressPolicy, "ingressPolicy"))
 	}
 
+	if s.Scale != nil {
+		if s.Scale.Minimum != nil && *s.Scale.Minimum < int64(0) {
```

Comment:
move this into a Validate method on the Scale struct. That will help the tests since the scale values can be tested independent of the rest of the spec.

Reply:
Moved these and also broke the Scale validation tests out to their own test. LMK if I've gone too far.